tensorflow_probability/python/distributions/pert.py

```diff
@@ -0,0 +1,164 @@
+"""The PERT distribution class"""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import tensorflow.compat.v2 as tf
+from tensorflow_probability.python.distributions import Beta
+from tensorflow_probability.python.bijectors import AffineScalar
+from tensorflow_probability.python.distributions import transformed_distribution
+from tensorflow_probability.python.internal import assert_util
+from tensorflow_probability.python.internal import dtype_util
+from tensorflow_probability.python.internal import prefer_static
+from tensorflow_probability.python.internal import tensor_util # pylint: disable=g-direct-tensorflow-import
+
+__all__ = [
+    'PERT',
+]
+
+class PERT(transformed_distribution.TransformedDistribution):
+  """Modified PERT distribution:
+  PERT distribution is a loc-scale family of Beta distribution intended to
+  model expert prediction on arbitrary real interval bounded by MAX and MIN
+  values set by the user, along with MODE to indicate the expert's
+  most frequent prediction.
+
+  The shift and scale factor is given by:
+
+  ```
+  shift = MIN
+  scale = MAX - MIN
+  ```
+
+  Modified PERT is a generalized version of PERT that has an additional
+  parameter, smoothness (g).
+
+  Smoothness determines the properties of Beta distribution:
+
+  ```
+  concentration1 = 1 + g * (MODE - MIN)/(MAX - MIN)
+  concentration0 = 1 + g * (MAX - MODE)/(MAX - MIN)
+  mean = (MIN + g * MODE + MAX)/(g + 2)
+  ```
+
+  PERT distribution is obtained when g = 4.
+
+  For general reference: https://en.wikipedia.org/wiki/PERT_distribution
+  """
+  def __init__(self,
+               mini,
```

Comment:
let's rename these to match the triangular distribution (low, high, peak [though mode is kind of nice too])
https://github.com/tensorflow/probability/blob/master/tensorflow_probability/python/distributions/triangular.py#L95
you might even mention PERT being "like Triangular, but smoother" or something along those lines.

Reply:
Done.